bokeh/io.py

```diff
@@ -311,14 +311,14 @@ def _show_with_state(obj, state, browser, new):
     elif state.server_enabled:
         _show_server_with_state(obj, state, new, controller)
 
-    if state.file:
+    else:
         _show_file_with_state(obj, state, new, controller)
```

Comment:
I think this is correct, because previously, if no filename was given, it would just ... do nothing.